docs/tutorials/traefik-proxy.md

```diff
@@ -0,0 +1,98 @@
+# Configuring ExternalDNS to use the Traefik Proxy Source
+
+This tutorial describes how to configure ExternalDNS to use the Traefik Proxy source.
+It is meant to supplement the other provider-specific setup tutorials.
+
+## Manifest (for clusters without RBAC enabled)
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        # update this to the desired external-dns version
+        image: k8s.gcr.io/external-dns/external-dns:v0.12.2
```

Comment:
I still have to provide a way to upgrade things in docs, but please bump it to v0.13.3 which is likely the release that will contain this feature.